grails-app/i18n/messages.properties

```diff
@@ -167,6 +167,7 @@ receiverDeployment.recoveryDate.validator.error=Recovery date must be after depl
 animalRelease.noTaggings=Animal release must have at least one tagging.
 animalRelease.noSpeciesOrAnimal=Species or existing animal must be specified.
 animalRelease.captureLocality.nullable=Capture locality must be specified.
+animalRelease.noSpecies=Existing species must be specified.
```

Comment:
@xhoenner  Let me know if the error message (animalRelease.noSpecies) needed to be updated or if it can be replaced by animalRelease.noSpeciesOrAnimal